web/packages/teleterm/src/ui/Search/searchResult.ts

```diff
@@ -27,7 +27,7 @@ type ResourceSearchResultBase<
   score: number;
 };
 
-export type SupportedResourceType = 'kubes' | 'servers' | 'databases';
+export type SupportedResourceType = 'kube_cluster' | 'node' | 'db';
```

Comment:
Could we use the `Pick` utility type to document that it's a subset of the type from unified resources?

Reply:
That's a good idea, btw I had to use `Extract` utility type to make a subset of union (`Pick` doesn't work).